app/tevm-tools/components/ui/use-toast.ts

```
@@ -0,0 +1,194 @@
+
+  return {
+    id: id,
+    dismiss,
+    update,
+  }
+}
+
+function useToast() {
+  const [state, setState] = React.useState<State>(memoryState)
+
+  React.useEffect(() => {
```

Biome (1.9.4)

[error] This hook specifies more dependencies than necessary: state

This dependency can be removed from the list.


(lint/correctness/useExhaustiveDependencies)